src/app/features/product/components/add-to-cart-button/add-to-cart-button.component.html

```diff
@@ -1,3 +1,3 @@
 <button type="button" (click)="handleAddToCart($event)" [disabled]="isDisabled">
-  <span class="material-symbols-outlined">shopping_cart</span>{{ text }}
+  <span class="material-symbols-outlined">shopping_cart</span>{{ `Add${isDisabled ? 'ed' : ''} to cart` }}
 </button>
```

Comment:
You can also write it like this:
```suggestion
  <span class="material-symbols-outlined">shopping_cart</span>Add{{ isDisabled ? 'ed' : '' }} to cart
</button>
```

Reply:
Done